packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -25,11 +26,12 @@ import { SidePanel } from './SidePanel';
 
 const useDraftOutbound = () => {
   const { id } = useParams();
+  const { api } = useOmSupplyApi();
 
   const { draft, save, dispatch, state } = useDocument(
     ['invoice', id],
     reducer,
-    OutboundShipmentDetailViewApi
+    getOutboundShipmentDetailViewApi(api)
```

Comment:
These are changed to pass in the api so that if we can change headers on the single instance etc